### ISSUE TITLE
Dash Table Width not correct

### ISSUE THREAD

**Issue by alejandrovalde** [facultyai/dash-bootstrap-components#372]
- dash version: 1.12.0
- dash-bootstrap-components version: 0.10.0
- dash-table: 4.7.0

### What is happening?

When using a `dash_table.DataTable` with `external_stylesheets=[dbc.themes.BOOTSTRAP]` or other Bootstrap Themes - the dash table won't show properly.

<img width="1280" alt="Screenshot 2020-05-12 at 18 42 58" src="https://user-images.githubusercontent.com/65018486/81721617-75b8ee00-9480-11ea-81f7-42f8a71a63bb.png">

When using a `dash_table.DataTable` with other css style sheets, ex: `external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']` 
<img width="1280" alt="Screenshot 2020-05-12 at 18 45 49" src="https://user-images.githubusercontent.com/65018486/81721864-d21c0d80-9480-11ea-8bac-0a98e1052e9b.png">

Both examples are with the layout:
`app.layout = html.Div([
    dash_table.DataTable(
        id = 'table',
        columns = [
        ],
        data = [
        ],
        style_cell={'textAlign': 'right'},
        fixed_rows={'headers': True},
        style_table={'height': 400},
        row_selectable="multi",
        selected_rows=[],
    )
])`

Is there a way to fix this?

**Comment by tcbegley** [comment]
Hi @alejandrovalde 

This is likely because `DataTable` and Bootstrap both use a CSS class called `row`, which clashes a bit. The easiest fix is to remove the changes Bootstrap makes to the `row` class within the data-table by writing some custom CSS like this

```css
.dash-table-container .row {
  display: block;
  margin: 0;
}
```

If you place that within your `assets/` folder it should improve the situation, let me know if not.

**Comment by ExperimentalHypothesis** [comment]
i actually have the same problem and it seems not possible to style it properly. would you be wiling to upload a simple demo of dash_table with bootstrap somewhere to see it?

**Comment by alejandrovalde** [comment]
My way to fix this was the following:

Download the .css class file from https://bootswatch.com/ that you want. The default BOOTSTRAP theme is here https://bootswatch.com/default/

Then add to the .css file the custom CSS @tcbegley posted above:

`.dash-table-container .row {
  display: block;
  margin: 0;
}`

![Screenshot 2020-05-17 at 17 45 22](https://user-images.githubusercontent.com/65018486/82153176-37e40d00-9866-11ea-9a46-f65747e3ca58.png)

Place it in a folder named 'assets' or as you wish, and use it as your `external_stylesheets`.

`app = dash.Dash(__name__, suppress_callback_exceptions=True, external_stylesheets=['assets/bootstrap.css'])`

**Comment by tcbegley** [comment]
This has been fixed in the latest version of Dash (>=1.18.1).

```
pip install -U dash
```

I've also made a note of the problem in the new [FAQ section](https://dash-bootstrap-components.opensource.faculty.ai/docs/faq/) in the docs for those stuck on older versions.
